.bumpversion.cfg

```diff
@@ -1,9 +1,6 @@
 [bumpversion]
-current_version = 0.16.7
+current_version = 0.17.0
 commit = True
-# The below regex details an acceptable version number by naming the groups (major, minor, patch, prerel, and
```

Comment:
why was this removed?

Reply:
the cli removed it, none of these changes are manual